docs/guides/deleting-series.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Deleting Series"
+linkTitle: "Deleting Series"
+weight: 3
+slug: deleting-series
+---
+
+:warning: This feature is currently experimental and is only supported for Chunks storage. :warning:
+
+Cortex supports deletion of series using [Prometheus compatible API](https://prometheus.io/docs/prometheus/latest/querying/api/#delete-series).
+It however does not support [Prometheuses Clean Tombstones](https://prometheus.io/docs/prometheus/latest/querying/api/#clean-tombstones) API by trading it in for another feature for allowing cancellation of delete requests.
+
+### How it works
+
+A new service called `purger` is added which handles deletion APIs and does the processing of the requests.
+To store the requests, and some additional information while performing deletion it requires configuring an index and object store respectively for it.
+`purger` can be configured using the config defined [here](../configuration/config-file-reference.md#purger_config)
+
+#### Requesting Deletion
```

Comment:
Which service should we hit? Can we include some curl commands? Specifically, we should also include the `X-Scope-OrgId` header, etc.